requirements/base.txt

```diff
@@ -8,26 +8,27 @@ amqp==1.4.9               # via kombu
 anyjson==0.3.3            # via kombu
 billiard==3.3.0.23        # via celery
 celery==3.1.26.post2      # via django-celery, edx-celeryutils
-certifi==2019.11.28       # via requests
+certifi==2020.4.5.1       # via requests
 chardet==3.0.4            # via requests
-django-celery==3.3.1      # via edx-celeryutils, super-csv
+django-celery==3.3.1      # via -r requirements/base.in
 django-crum==0.7.5        # via super-csv
-django-model-utils==3.0.0 ; python_version == "2.7"
-django==1.11.26
-djangorestframework==3.9.4  # via super-csv
-edx-celeryutils==0.3.1    # via super-csv
-edx-opaque-keys==2.0.1
+django-model-utils==4.0.0  # via -r requirements/base.in, edx-celeryutils, super-csv
+django==2.2.12            # via -c requirements/constraints.txt, -r requirements/base.in, django-celery, django-crum, django-model-utils, djangorestframework, edx-celeryutils, jsonfield2, super-csv
+djangorestframework==3.11.0  # via super-csv
+edx-celeryutils==0.5.0    # via super-csv
+edx-opaque-keys==2.0.2    # via -r requirements/base.in
 future==0.18.2            # via edx-celeryutils
-idna==2.8                 # via requests
-jsonfield==2.0.2          # via edx-celeryutils
+idna==2.9                 # via requests
+jsonfield2==4.0.0.post0   # via edx-celeryutils
```

Comment:
jsonfield2 > 3.0.2 has dropped support for python 3.5. edx-celeryutils has this constraint but still it is pulling latest version. We are good as long as it is not causing trouble?

https://pypi.org/project/jsonfield2/

Reply:
jsonfield2 version `3.0.3` is the last version to support `python3.5` so pinning `jsonfield2<=3.0.3`.